doit/runner.py

```diff
@@ -113,7 +113,6 @@ def select_task(self, node, tasks_dict):
 
             # overwrite with effective verbosity
             task.overwrite_verbosity(self.stream)
-            task.verbosity = self.stream.effective_verbosity(task.verbosity)
```

Comment:
These two lines of code are doing the exact same thing, so I deleted one of the lines.